seb_openedx/settings/common.py

```diff
@@ -18,4 +18,4 @@ def plugin_settings(settings):
     Defines seb_openedx settings when app is used as a plugin to edx-platform.
     See: https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/plugins/README.rst
     """
-    pass
+    settings.IS_CMS = settings.__name__.startswith('cms')
```

Comment:
I think a cleaner solution is to separate the middlewares into a cms middleware and a lms one. We are overcomplicating this a lot, and we are gaining only the ability to auto add a key to the other_configs dict. A user configuring its exam to work on SEB is most definitely a power user and can be trusted to add a specific key to a field in studio.

Reply:
Done